config/mattermost-push-proxy.json

```diff
@@ -4,6 +4,7 @@
     "ThrottleMemoryStoreSize":50000,
     "ThrottleVaryByHeader":"X-Forwarded-For",
     "EnableMetrics": false,
+    "ProxyServer": "",
```

Comment:
Just curious, do we actually want an explicit config setting here, or is just obeying `HTTP_PROXY`/`HTTPS_PROXY` environment variables sufficient? I'm not actually sure what we do in the server, but I guess it makes the most sense to follow the same pattern here.